File: backend/src/Dtos/PatchAppointmentDto.cs
using LogopedicBackend.Enums;

namespace LogopedicBackend.Dtos;

public class PatchAppointmentDto
{
    public DateTimeOffset? StartTime { get; set; }
    public int? DurationInMinutes { get; set; }
    public AppointmentType? Type { get; set; }
    public AppointmentStatus? Status { get; set; }
    public int? PatientId { get; set; }
}

File: frontend/src/app/core/models/PatchAppointmentDto.ts
import { AppointmentType } from './AppointmentType';
import { AppointmentStatus } from './AppointmentStatus';

export interface PatchAppointmentDto {
    startTime?: Date;
    durationInMinutes?: number;
    type?: AppointmentType;
    status?: AppointmentStatus;
    patientId?: number;
}
